pagopa/io-app
# feat(IDPay): [IODPAY-171] IDPay initiative unsubscription

## Short description
This PR adds the ability to unsubscribe from an IDPay initiative and filters out the unsubscribed initiatives from the citizen wallet.

https://user-images.githubusercontent.com/6160324/230343654-1b255961-0ee9-41e7-9211-8bf0fd51a96d.mp4

## List of changes proposed in this pull request
- Added button in the initiative details screen to start the unsubscription flow (this is temporary)
- Added new routes to the navigation stack
- Added screens, components and XState machine for the unsubscription flow
- Add `idPayWalletSubscribedInitiativeListSelector` to allow wallet screen to display only subscribed initiatives

## How to test
- Start the `io-dev-server`
- Navigate to the details screen of an IDPay initiative
- Tap on the unsubscribe button at the end of the screen
- Follow the instruction displayed in the screen to complete the unsubscription
- If unsubscription is successfull, verify that the initiative is no more displayed in the wallet
